pkg/templates/charts/toggle/cluster-manager/templates/cluster-manager.yaml

```diff
@@ -79,13 +79,6 @@ spec:
             - ALL
           privileged: false
           runAsNonRoot: true
-          readOnlyRootFilesystem: true
-        volumeMounts:
-        - name: tmpdir
-          mountPath: /tmp
-      volumes:
-        - name: tmpdir
-          emptyDir: { }
```

Comment:
I'm not sure we want to remove these SCC fields from the `cluster-manager` deployment. For now, lets keep this PR on hold until we can further verify.